test/integration/admin/admin-v1/Urls.test.ts

```diff
@@ -0,0 +1,209 @@
+import { MessageType } from '../../../../src/shared/util/messages'
+import { ADMIN_API_V1_URLS } from '../../config'
+import {
+  DATETIME_REGEX,
+  createIntegrationTestAdminUser,
+  createIntegrationTestUser,
+  deleteIntegrationTestUser,
+  generateRandomString,
+} from '../../util/helpers'
+import { postJson } from '../../util/requests'
+
+async function createLinkUrl(
+  link: {
+    shortUrl?: string
+    longUrl?: string
+    email?: string
+  },
+  apiKey: string,
+) {
+  const res = await postJson(ADMIN_API_V1_URLS, link, undefined, apiKey)
+  return res
+}
+
+/**
+ * Integration tests for URLs.
+ */
+describe('Url integration tests', () => {
```

Comment:
nit: would it be good to specify that these integration tests are for the admin API?

Reply:
indeed! renamed in https://github.com/opengovsg/GoGovSG/pull/2213/commits/53bc68a3fd2ed17aad2f6189e42162ce9e57f832 thanks